WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabViewModel.swift

```diff
@@ -37,6 +38,20 @@ class ReaderTabViewModel {
         bottomSheet.show(from: from, sourceView: sourceView, arrowDirections: .up)
     }
 
+    func presentManage(from: UIViewController) {
+        let tagsVC = ReaderTagsTableViewController(style: .grouped)
```

Comment:
It's probably just a matter of preference, but maybe shorthand names for variables are less readable? meaning tagsViewController is more readable than tagsVC?